RecFCCeeCalorimeter/src/components/PhotonIDTool.cpp

```diff
@@ -336,10 +338,12 @@ StatusCode PhotonIDTool::readMVAFiles(const std::string& mvaInputsFileName,
   debug() << "Output Node Name/Shape (" << m_ortSession->GetOutputCount() << "):" << endmsg;
   for (std::size_t i = 0; i < m_ortSession->GetOutputCount(); i++)
   {
-    // for old ONNX runtime version
-    // m_output_names.emplace_back(m_ortSession->GetOutputName(i, allocator));
-    // for new runtime version
+#if ORT_API_VERSION < 13
+    m_output_names.emplace_back(AllocatedStringPtr(m_ortSession->GetOutputName(i, allocator), allocDeleter).release());
+#else
     m_output_names.emplace_back(m_ortSession->GetOutputNameAllocated(i, allocator).get());
```

Comment:
should this .get() be a .release() instead?

Reply:
Yes, almost certainly. Otherwise we will have a dangling pointer to a temporary. Sorry about that. We will be leaking these pointers at the end though then, so we will probably have to cleanup in the destructor.